examples/surface_texture_and_colors.py

```diff
@@ -0,0 +1,56 @@
+"""
+Surface with texture and vertex_colors
+==========
```

Comment:
Nitpicking but som RST parser really want the same length 

```suggestion
Surface with texture and vertex_colors
======================================
```